src/variorum/Intel/Haswell_3F.c

```diff
@@ -144,52 +149,95 @@ int fm_06_3f_get_features(void)
     printf("Running %s\n", __FUNCTION__);
 #endif
```

Comment:
Is this the right indentation for `#endif`? Didn't take a look at the raw file (too late here).

Reply:
Not sure if it is rendering incorrectly. Other `ifdefs` and `endifs` look different on my review screen..